src/load/create-postgres-tables.js

```diff
@@ -2,11 +2,11 @@
 const async = require('async');
 
 // prepare postgresql connection to the database
-const pg = require('../lib/postgresQL')(require('../config/pgconfig'));
+const pg = require('../lib/utils/postgresQL')(require('../config/pgconfig'));
 
 // prepare commands we want to execute
 let commands = [
-    'CREATE TABLE IF NOT EXISTS client_activity (id serial PRIMARY KEY, uuid varchar NOT NULL, provider varchar NOT NULL, url varchar NOT NULL, referrer varchar NOT NULL, visitedOn timestamp with time zone);',
+    'CREATE TABLE IF NOT EXISTS client_activity (id serial PRIMARY KEY, uuid varchar NOT NULL, provider varchar NOT NULL, url varchar NOT NULL, referrer varchar NOT NULL, visitedOn timestamp with time zone NOT NULL);',
```

Comment:
Personally, I would prefer if there were no long lines - it makes it prettier to look at and easier to do a review on :) (It does not have to be 80 characters long, it would be nice if it you can see it on github nicely.)